plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
+                        required=True, help="Label Selector")
+    return parser
+
+
+check_mongodb_cmd = ("bash", "-c",
+                     "mongo 127.0.0.1/admin "
+                     "-u \"admin\" -p \"$MONGODB_ADMIN_PASSWORD\" "
+                     "--eval=\"rs.status().myState\" --quiet")
```

Comment:
- If you start the list of arguments in the next line, you need less indentation
- Using single quotes allows us to remove backslashes escaping the double quotes
- IMHO mixing multiple arguments (comma separated) and the string concatenation trick (no commas) seem to confuse readers (at least me). It makes one think why are there spaces in the end of the strings, and why are there no commas line in the previous line... and some time later, ah! Implicit string concatenation!

I'd suggest a bit longer line, but clear intention:

```python
check_mongodb_cmd = (
    "bash", "-c",
    'mongo 127.0.0.1/admin -u "admin" -p "$MONGODB_ADMIN_PASSWORD" --eval="rs.status().myState" --quiet'
)
```

WDYT?